detection-rules/attachment_microsoft_image_lure_qr_code.yml

```
@@ -17,6 +16,11 @@ source: |
               any(.scan.strings.strings, regex.icontains(.,  'scan|camera'))   
               and any(.scan.strings.strings, regex.icontains(.,  '\bQR\b|Q\.R\.|barcode'))
           )
+          or (
+              any(file.explode(.), .scan.qr.type == "url"
+              and any(recipients.to, strings.icontains(..scan.qr.data, .email.email))
```

```suggestion
                  // recipient email address is present in the URL, a common tactic used in credential phishing attacks 
                  and any(recipients.to, strings.icontains(..scan.qr.data, .email.email)
              )
```
inside the filexplode so this should be indented